packages/ia-components/sandbox/details/Search.jsx

```diff
@@ -19,31 +21,32 @@ class SearchSwitcher extends IAReactComponent {
       <div className="topinblock">
         <div className="hidden-xs hidden-sm">
           <div className="sort-by">
-            <I18nSpan en="SORT BY"/>
+            <I18nSpan en="SORT BY" />
           </div>
-          {/*--TODO-DETAILS this dropdown doesnt reorder, test other UI elements in vicinity as well see https://github.com/internetarchive/dweb-archive/issues/15--*/}
-          <span className="big-label blue-pop">{/*TODO-ISSUE dweb-archive#57 remove relevance on Collections*/}
-            {(!this.props.identifier) ? // Dont show on collections
-              <AnchorSearch className="ikind stealth in" data-id="relevance" query={this.props.query}><I18nSpan en="RELEVANCE"/></AnchorSearch>
+          {/* --TODO-DETAILS this dropdown doesnt reorder, test other UI elements in vicinity as well see https://github.com/internetarchive/dweb-archive/issues/15--*/}
+          <span className="big-label blue-pop">{/* TODO-ISSUE dweb-archive#57 remove relevance on Collections */}
+            {(!this.props.identifier) // Dont show on collections
+              ? <AnchorSearch className="ikind stealth in" data-id="relevance" query={this.props.query}><I18nSpan en="RELEVANCE" /></AnchorSearch>
               : undefined}
-            {(!this.props.identifier) ? // Dont show on collections
-              <div className="iconochive-dot ikind-sep"></div>
+            {(!this.props.identifier) // Dont show on collections
+              ? <div className="iconochive-dot ikind-sep" />
               : undefined}
```

Comment:
this can be in the same block. `null` is the appropriate value to send if you don't want DOM to be drawn.

Also, you can use the `!this.props.identifier && <DOM>` shortcut - also for better readability

Reply:
Change made to use null because I understand that method,  (note the use of undefined is in old code before I knew about null, only change here was lint-driven formating to drop the "?" down.

Reply:
Can you explain that second construct. I believe `!this.props.identifier && <DOM>` returns `false` and I thought you had to send `null` to avoid drawing DOM. If you are sure that is valid I will make that change as I edit code in future.